app.rb

```diff
@@ -1,9 +1,41 @@
 require 'sinatra/base'
+require_relative './lib/peep'
 
 class Chitter < Sinatra::Base
+  require 'sinatra/reloader' if development?
+
+  configure :development do
+    register Sinatra::Reloader
+  end
+
+  enable :sessions
+  attr_reader :peep_date
+
   get '/test' do
     'Test page'
   end
 
+  get '/' do
+    @peeps = Peep.all
+    erb(:index)
+  end
+
+  get '/new_peep' do
+    erb(:new_peep)
+  end
+
+  post '/post_peep' do
+    time = Time.new
```

Comment:
Normally, it's considered best practices to keep our controllers as short and concise as possible (i.e. `skinny controllers`). In this case, I would refactor this route and move the logic related with the `time` to the Model instead. How does this sound?